src/autolabel/labeler.py

```diff
@@ -334,20 +339,29 @@ def plan(
         print(f"{prompt_list[0]}")
         console.rule()
 
+    async def async_run_transform(
+        self, transform: BaseTransform, dataset: AutolabelDataset
+    ):
+        transform_outputs = [
+            transform.apply(input_dict) for input_dict in dataset.inputs
+        ]
+        outputs = await gather_async_tasks_with_progress(
+            transform_outputs,
+            description=f"Running transform {transform.name()}...",
+            console=console,
+        )
+        output_df = pd.DataFrame.from_records(outputs)
+        final_df = pd.concat([dataset.df, output_df], axis=1)
+        dataset = AutolabelDataset(final_df, self.config)
```

Comment:
we should probably add this as a util function in autolabelDataset so we can change this if needed in the future. I missed this when creating the base transform